Rocket.Chat.Shared/Controllers/Chat/ChatViewController.swift

```diff
@@ -452,7 +452,7 @@ public class ChatViewController: SLKTextViewController, AuthManagerInjected, Soc
                 let indexPaths = self.dataController.insert(objs)
                 collectionView.insertItems(at: indexPaths)
                 if self.messageCellStyle == .bubble {
-                    collectionView.reloadItems(at: indexPaths.filter { $0.row > 0 }.map { IndexPath(row: $0.row - 1, section: $0.section) })
+                    collectionView.reloadItems(at: indexPaths.filter { $0.row > 0 }.map { IndexPath(row: $0.row - 1, section: $0.section) }.filter { !indexPaths.contains($0) })
```

Comment:
This line is very complex... can you simplify this code or split in more than 1 line? :-)